README.md

```diff
@@ -13,21 +12,45 @@ Teams spend significant time building parts of the design system (colors, variab
 [![code style: prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square)](https://github.com/prettier/prettier)
 ![license](https://img.shields.io/github/license/mashape/apistatus.svg)
 
+:star: We appreciate your star, it helps!
+
+## Key features & benefits
+### Unopionated
+These days, there are a million and one great ways to build your next Web site or application. So, we have made being unopinionated the center of every decision made in FAST-DNA. 
```

Comment:
here you use 'Web site" and in the first paragraph you use "websites".  Should we be using one, the other, or both?

Reply:
Fixed.